app/tests/integration/nimbus/test_mobile_integration.py

```diff
@@ -0,0 +1,128 @@
+import json
+import os
+from pathlib import Path
+
+import pytest
+from nimbus.models.base_app_context_dataclass import BaseAppContextDataClass
+from nimbus.utils import helpers
+
+nimbus = pytest.importorskip("nimbus_rust")
+
+
+def locale_number_loader(locales=None):
+    locale_list = []
+    path = Path().resolve()
+    path = str(path)
+    path = path.strip("/tests/integration/nimbus")
+    path = f"/{path}/experimenter/base/fixtures/locales.json"
+    with open(path) as file:
+        data = json.loads(file.read())
+        for locale in locales:
+            for item in data:
+                if locale in item["fields"]["code"][:2]:
+                    locale_list.append(item["pk"])
+    return locale_list
+
+
+def client_info_list():
+    with open("nimbus/app_contexts.json") as file:
+        return json.load(file)["query_result"]["data"]["rows"]
+
+
+@pytest.fixture
+def load_app_context():
+    def _load_app_context_helper(context):
+        base_app_context = BaseAppContextDataClass.from_dict(context)
+        return nimbus.AppContext(
+            app_id=base_app_context.app_id,
+            app_name=base_app_context.app_name,
+            channel=base_app_context.channel,
+            app_version=base_app_context.app_version,
+            app_build=base_app_context.app_build,
+            architecture=base_app_context.architecture,
+            device_manufacturer=base_app_context.device_manufacturer,
+            device_model=base_app_context.device_model,
+            locale=base_app_context.locale,
+            os=base_app_context.os,
+            os_version=base_app_context.os_version,
+            android_sdk_version=base_app_context.android_sdk_version,
+            debug_tag=base_app_context.debug_tag,
+            installation_date=base_app_context.installation_date,
+            home_directory=base_app_context.home_directory,
+            custom_targeting_attributes=base_app_context.custom_targeting_attributes,
+        )
+
+    return _load_app_context_helper
+
+
+@pytest.fixture(name="ar_units")
+def fixture_ar_units():
+    return nimbus.AvailableRandomizationUnits(None, 0)
+
+
+@pytest.fixture(name="sdk_client")
+def fixture_sdk_client(ar_units):

```

Comment:
What's `ar_units`?

Reply:
um.... @tarikeshaq ? I know it was needed to create the rust client

Reply:
We probably want to include the full name here 😄 It's the `available_randomization_units` - it helps with randomization, in the case a bucket configuration wants to randomize using a client_id (useful in some tests too) 

if the experiments all use `nimbus_id` as the `randomizationUnit`, we can just pass a default value (i.e construct it with None as the `client_id` as Ben is doing here!)

Reply:
Thanks Tarik...I added a docstring